src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs

```diff
@@ -29,8 +30,15 @@ public class DisplayVersion
             string additionalInfo,
             bool isValidVersion = true,
             bool isCurrentInstalled = false,
-            bool autoReferenced = false)
+            bool autoReferenced = false,
+            string versionFormat = "N")
         {
+            if (versionFormat == null)
```

Comment:
This could be null :) https://github.com/NuGet/NuGet.Client/pull/1519/files#diff-c20d2bf504a1dc14b412134958fb54e9R22